pkg/splunk/enterprise/util.go

```diff
@@ -31,17 +36,97 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	enterpriseApi "github.com/splunk/splunk-operator/api/v3"
 	splclient "github.com/splunk/splunk-operator/pkg/splunk/client"
 	splcommon "github.com/splunk/splunk-operator/pkg/splunk/common"
 	splctrl "github.com/splunk/splunk-operator/pkg/splunk/controller"
 	splutil "github.com/splunk/splunk-operator/pkg/splunk/util"
+
+	// Used to move files between pods
+	_ "unsafe"
 )
 
 // kubernetes logger used by splunk.enterprise package
 //var log = logf.Log.WithName("splunk.enterprise")
 
+var operatorResourceTracker *globalResourceTracker = nil
+
+// initialize operator level context
+func init() {
+	fmt.Printf("init is called here\n")
+	defer fmt.Printf("init ends here\n")
```

Comment:
were these intentional ?

Reply:
yes was used in debugging, removed for now